src/edu/csus/ecs/pc2/core/imports/LoadICPCTSVData.java

```diff
@@ -112,8 +115,8 @@ public boolean loadFiles(String filename, boolean sendToServer, boolean useConfi
 
             if (useConfirmGUI){
                 
-                String nl = System.getProperty("line.separator");
-                String message = "Add " + nl + accounts.length + " accounts and " + nl + groups.length + " groups?";
+                String NL = System.getProperty("line.separator");
+                String message = "Update/Add " + NL + accounts.length + " team accounts and " + NL + groups.length + " groups?";
```

Comment:
I don't understand why you went to the trouble to add "NL" to the Constants class, but then here invoked System.getProperty("line.separator").  Not a blocker; just seems inconsistent...

Reply:
A couple of reasons, NL is shorter and easier to read the string.  
NL is a refactoring change.